test/e2e/csv_e2e_test.go

```diff
@@ -714,39 +725,12 @@ func TestCreateCSVRequirementsMetCRD(t *testing.T) {
 	_, err = c.CreateClusterRoleBinding(&nonResourceClusterRoleBinding)
 	require.NoError(t, err, "could not create ClusterRoleBinding")
 
-	cleanupCSV, err := createCSV(t, c, crc, csv, testNamespace, true, false)
-	require.NoError(t, err)
-	defer cleanupCSV()
-
-	fmt.Println("checking for deployment")
-	// Poll for deployment to be ready
-	err = wait.Poll(pollInterval, pollDuration, func() (bool, error) {
```

Comment:
is there a reason we removed the deployment check?

(It's fine as long as there's another test somewhere that can verify that if the CSV succeeded, the deployment is actually running)

Reply:
The `ocs` e2e tests poll for deployment status, so nevermind